verifier/src/report_body.rs

```diff
@@ -244,18 +244,55 @@ impl IntoVerificationError for MiscellaneousSelect {
 }
 
 /// Verifier for ensuring [`MrEnclave`] values are equivalent.
+///
+/// The Intel SDK docs refer to this as "Strict Enclave Modification Policy"
```

Comment:
Not sure if we want to add a similar key monicker to the `MrEnclaveVerifier` and friends.